From 3b18355415f437c59809ff94f1c365a1cdfaa523 Mon Sep 17 00:00:00 2001
From: kiamvdd <kiamvdd@outlook.com>
Date: Fri, 26 Dec 2025 21:14:54 +0100
Subject: [PATCH] Add Lenovo Y50-70 touchpad to 60-evdev.hwdb

(cherry picked from commit 506b790d71aff5cd8612c3dd256872341db94ab9)

Resolves: RHEL-72702
---
 hwdb.d/60-evdev.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 57b349bff3..e09c221d15 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -704,6 +704,13 @@ evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnLENOVO:*pvrLenovoideapadY700-14ISK:*
  EVDEV_ABS_35=::27
  EVDEV_ABS_36=::29
 
+#Lenovo Y50-70
+evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnLENOVO:*pvrLenovoY50-70**
+ EVDEV_ABS_00=164:3918:37
+ EVDEV_ABS_01=120:1935:26
+ EVDEV_ABS_35=164:3918:37
+ EVDEV_ABS_36=120:1935:26
+
 # Lenovo Ideapad 310S-14ISK
 evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnLENOVO:*pvrLenovoideapad310S-14ISK:*
  EVDEV_ABS_00=113:3960:37
